src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -179,9 +208,11 @@
         </service>
         <service id="form.type_extension.repeated.validator" class="Symfony\Component\Form\Extension\Validator\Type\RepeatedTypeValidatorExtension">
             <tag name="form.type_extension" extended-type="Symfony\Component\Form\Extension\Core\Type\RepeatedType" />
+            <deprecated>The "%service_id%" service is deprecated since Symfony 3.1 and will be removed in 4.0.</deprecated>
```

Comment:
@stof I don't think hey have to since they don't have strict dependencies.

See https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Form/Extension/Validator/ValidatorExtension.php#L54.

Right ?